ChaosMod/Effects/db/Player/PlayerHasGravity.cpp

```diff
@@ -53,19 +52,12 @@ static void OnTick()
 			}
 			float forceDistance = min(max(0.f, (startDistance - distance)), maxForceDistance);
 			float force = (forceDistance / maxForceDistance) * maxForce;
-			APPLY_FORCE_TO_ENTITY(entity, 3, (entityCoord.x - playerCoord.x) * -1.f, (entityCoord.y - playerCoord.y) * -1.f, (entityCoord.z - playerCoord.z) * -1.f, 0, 0, 0, false, false, true, true, false, true);
+			APPLY_FORCE_TO_ENTITY(entity, 3, (entityCoord.x - playerCoord.x) * -3.f, (entityCoord.y - playerCoord.y) * -3.f, (entityCoord.z - playerCoord.z) * -3.f, 0, 0, 0, false, false, true, true, false, true);
 		
 			if (IS_ENTITY_A_MISSION_ENTITY(entity))
 			{
 				SET_ENTITY_INVINCIBLE(entity, true);
 			}
-
-			if (--count <= 0)
-			{
-				WAIT(0);
```

Comment:
This was to prevent crashes and performance lags

Reply:
Aware, this was specifically to increase crashes and make the force of the gravity bigger.